lib/src/runtime/composables/useAsyncStoryblok.js

```diff
@@ -1,13 +1,13 @@
-import { useStoryblokApi, useStoryblokBridge } from "@storyblok/vue";
 import { useAsyncData, useState, onMounted } from "#imports";
+import { useStoryblokApi, useStoryblokBridge, ISbStoryData } from '@storyblok/vue';
```

Comment:
This could probably be @SebbeJohansson:

```suggestion
import { useStoryblokApi, useStoryblokBridge } from '@storyblok/vue';
import type { ISbStoryData } from '@storyblok/vue';
```

What do you think?